Configurations/10-main.conf

```diff
@@ -192,10 +193,11 @@ my %targets = (
         # failures [at least] in 32-bit build.
         inherit_from     => [ "solaris-common", asm("x86_elf_asm") ],
         cc               => "gcc",
-        cflags           => add_before(picker(default => "-Wall -DL_ENDIAN",
+        cflags           => add_before(picker(default => "-Wall",
                                               debug   => "-O0 -g",
                                               release => "-O3 -fomit-frame-pointer"),
                                        threads("-pthread")),
+        cppflags         => add("-DL_DEBIAN"),
```

Comment:
I think this should be `-DL_ENDIAN`.

Reply:
Heh, this is a typical case of "brain do one thing, fingers another"...  took me three hard looks before I saw the difference.  Will fix, thanks!